File: src/main/java/net/igneo/icv/entity/weapon/FireRing/FireRingRenderer.java
package net.igneo.icv.entity.weapon.FireRing;

import net.minecraft.client.renderer.entity.EntityRendererProvider;
import software.bernie.geckolib.renderer.GeoEntityRenderer;

public class FireRingRenderer extends GeoEntityRenderer<FireRingEntity> {
    
    public FireRingRenderer(EntityRendererProvider.Context renderManager) {
        super(renderManager, new FireRingModel());
    }
}

File: src/main/java/net/igneo/icv/entity/weapon/fireRing/FireRingRenderer.kt
package net.igneo.icv.entity.weapon.fireRing

import net.minecraft.client.renderer.entity.EntityRendererProvider
import software.bernie.geckolib.renderer.GeoEntityRenderer

class FireRingRenderer(renderManager: EntityRendererProvider.Context) :
    GeoEntityRenderer<FireRingEntity?>(renderManager, FireRingModel())
